docs-assets/screenshots/schema.js

```diff
@@ -2954,4 +2954,18 @@ export default {
             deviceScaleFactor: 3,
         },
     },
+    'panels/navigation/customizing-user-menu': {
+        url: 'top-bar',
+        selector: 'body > div.fi-layout.flex.min-h-screen.w-full.flex-row-reverse.overflow-x-clip > div.fi-main-ctn.w-screen.flex-1.flex-col.opacity-0 > div > nav > div > div > div.fi-dropdown-panel.absolute.z-10.w-screen.divide-y.divide-gray-100.rounded-lg.bg-white.shadow-lg.ring-1.ring-gray-950\\/5.transition.dark\\:divide-white\\/5.dark\\:bg-gray-900.dark\\:ring-white\\/10.\\!max-w-\\[14rem\\]',
+        viewport: {
+            width: 1080,
+            height: 640,
+            deviceScaleFactor: 3,
+        },
+        before: async (page) => {
+            await page.click('body > div.fi-layout.flex.min-h-screen.w-full.flex-row-reverse.overflow-x-clip > div.fi-main-ctn.w-screen.flex-1.flex-col.opacity-0 > div > nav > div > div > div.fi-dropdown-trigger.flex.cursor-pointer')
```

Comment:
I really REALLY appreciate the work you put into this, adding screenshots to the docs is a goal we have and it's great to see people investigate the screenshot system we have in order to contribute, as it scares a lot of people off

My only feedback here is to suggest that we add CSS classes to Filament so these selectors can be simplified, I don't want them to break if we change Tailwind classes etc.

Reply:
Once the PR is merged, the image will be hosted on GitHub & then you can run `npm run generate-docs` from the `filamentphp.com` repo to check it out, but I will do that for you when I deploy next